dpctl/tensor/libtensor/source/integer_advanced_indexing.cpp

```diff
@@ -1078,9 +1094,10 @@ usm_ndarray_put(dpctl::tensor::usm_ndarray dst,
             sycl::free(packed_ind_offsets, ctx);
         });
     });
+    host_task_events.push_back(put_generic_ev);
 
     return std::make_pair(
-        keep_args_alive(exec_q, {dst, py_ind, val}, {put_generic_ev}),
+        keep_args_alive(exec_q, {dst, py_ind, val}, {host_task_events}),
```

Comment:
I think `{host_task_events}` should become `host_task_events`. The function takes vector of events in the last argument.